tests/conftest.py

```diff
@@ -18,3 +21,13 @@ def global_mmcore(core):
     assert mmc == core
     mmc.loadSystemConfiguration(str(Path(__file__).parent / "test_config.cfg"))
     return mmc
+
+
+@pytest.fixture()
+def px_wdg(global_mmcore, qtbot: QtBot):
```

Comment:
unless you need to use this fixture in multiple different test files, it should be placed in the test file where it's being used.  (i.e.  move this to `test_table_pixel_size_widget`)